text/0000-lock-without-registry.md

```diff
@@ -0,0 +1,35 @@
+- Start Date: 2017-05-09
+- RFC PR:
+- Yarn Issue:
+
+# Summary
+
+The lockfile yarn.lock should not include the base registry (`https://registry.npmjs.org`).
+
+# Motivation
+
+In yarn.lock, the `resolved` field includes registry such as `https://registry.npmjs.org`.
+In China, most developers will set it to `https://registry.npm.taobao.org` for speed; but it seems slow for travis-ci and circleci.
+
+# Detailed design
+
+Replace the `resolved` by a `hash` field.
```

Comment:
Does `registry-replace` prevent me from leaking internal URLs into a `yarn.lock` file that gets publicly published?

You mention that it works at fetch time, so I wasn't sure if you were implying that `registry-replace` is used also for writing the full path to `yarn.lock`, even if your registries are configured differently in your `.npmrc` or `.yarnrc` files.